CHANGES/8081.feature

```diff
@@ -0,0 +1 @@
+Basic CRUD for Ansible collection remotes, use '-t collection' after remote to select type
```

Comment:
I tried `pulp ansible remote create -t collection --name myremote --url https://galaxy.ansible.com` and it says `Error: no such option: -t`

Reply:
This is on a49b6ba8a8b864e07cfa1b62db1f39cc508c3997

Reply:
I guess I worded it poorly, it's `pulp ansible remote -t collection create --name myremote --url https://galaxy.ansible.com`
Also, I'm not sure how --requirements-file is suppose to be inputed into collection remote create. I tried using the example on the workflows, but it didnt work.

Reply:
You can use `pulp -v <...>` in order to see the http requests made by the cli. This might help you debugging.